packages/@aws-cdk/aws-kms/lib/perms.ts

```diff
@@ -0,0 +1,28 @@
+// https://docs.aws.amazon.com/kms/latest/developerguide/key-policies.html
+
+export const ADMIN_ACTIONS = [
```

Comment:
Should we put this file in `lib/private`? I understand KMS doesn't use this convention, but you have to start somewhere 🙂.